src/annotations/components/EditAnnotationOverlay.tsx

```diff
@@ -0,0 +1,39 @@
+// Libraries
+import React, {FC} from 'react'
+
+// Components
+import AnnotationForm from 'src/annotations/components/annotationForm/AnnotationForm'
+
+// Types
+import {Annotation} from 'src/annotations/reducers/annotationReducer'
+
+const MOCK_ANNOTATION: Annotation = {
+  summary: 'I am an annotation',
+  message: 'Look at me go',
+  type: 'point',
+  timeStart: 'timeStart',
+  timeStop: 'timeStop',
+  streamID: 'anno2',
+  bucketName: 'defBuck',
+  measurement: 'annotations',
+  tags: {
+    _field: 'snacks',
+  },
+}
+
+const EditAnnotationOverlay: FC = () => {
+  // const handleSubmit = (annotation: Annotation): void => {
```

Comment:
dont keep around commented out code. you can get around the linting error by prepending an underscore to annotation